[Feature/admin recruitment]  recruitment admin. serializer-lecture 부분 수정

romiii35 opened the pull request:
## ✅ PR 요약
- 작업 요약: recruitment admin/serializer-lecture 부분 수정

## 📄 상세 내용
- [X] recruitment admin/serializer-lecture 부분 수정

## 🧪 PR Checklist
- [X] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [X] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

romiii35 committed:
💡 chore : recruitment admin. serializer-lecture 부분 수정

recruitment admin. serializer-lecture 부분 수정

2joonkim commented on apps/recruitment/serializers/recruitment_admin_serializer.py:
```diff
@@ -31,11 +31,13 @@ class Meta:
+    title = serializers.CharField()
+    instructor = serializers.CharField()
+    thumbnail_img_url = serializers.CharField()
+    url_link = serializers.CharField()
```
이거 지금 수정하신 이유가 application 부분의 시리얼라이저랑 겹쳐서 변경하신건가요?
component_name이 lectures로 진행했을때 의미도 애매하고 다른 lectures스키마 있을때 충돌납니다
차라리 클래스명을 바꿔서 진행하시는게 좋아보여요

romiii35 committed:
💡 chore : AdminRecruitmentLectureSerializer를 AdminRecruitmentLectureDetailSerializer로 변경

AdminRecruitmentLectureSerializer를 AdminRecruitmentLectureDetailSerializer로 변경

romiii35 committed:
💡 chore  : AdminRecruitmentLectureSerializer를 AdminRecruitmentLectureSummarySerializer로 변경

AdminRecruitmentLectureSerializer를 AdminRecruitmentLectureSummarySerializer로 변경
